### PR TITLE
Autoclose test figures

### PR DESCRIPTION
**Pull request by jl-wynen**
This fixes the warning that more than 20 figures have been opened.

Most changes are simply removing `plot` from `plot_helper.py` and calling `sc.plot` instead. The relevant change is in `conftest.py`.

### CODE REVIEW

File: tests/plotting/conftest.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import matplotlib.pyplot as plt
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def close_figures():
```
Comment (SimonHeybrock) [inline tests/plotting/conftest.py:6]: This fixture is now automatically used for all tests in the `plotting/` folder? But tests outside are not affected?

**Review by SimonHeybrock — APPROVED** [review]
Nice solution 👍 

Maybe @nvaytet should have a brief look?

File: tests/plotting/conftest.py (hunk 1 above)
Comment (jl-wynen) [reply tests/plotting/conftest.py:6]: As far I understand it, yes.

Comment (jl-wynen) [reply tests/plotting/conftest.py:6]: Checked by making `close_figures` raise an exception. That makes only plotting tests fail, everything else is unaffected.

**Pull request merged**